websites/S/Stadia/presence.ts

```diff
@@ -1,5 +1,5 @@
 const presence = new Presence({
-  clientId: "713114770584109150"
+  clientId: "844107770402701332"
 });
 const browsingStamp = Math.floor(Date.now() / 1000);
 let gameName: HTMLElement;
```

Comment:
Well about stadia - someone else is going to do it - so basically revert changes for now.